src/Document.php

```diff
@@ -1732,7 +1739,8 @@ public static function dropdown($options = [])
         $out .= "<span id='show_" . $p['name'] . "$rand'>";
         $out .= "</span>\n";
 
-        $params['rubdoc'] = 0;
+        $params['rubdoc'] = $p['rubdoc'] ?? 0;
+        $params['value'] = $p['value'] ?? 0;
```

Comment:
Defaiult values shouyld be declared at the top of the method, with other ones

Reply:
The `$params` are sent just before the call to `Ajax::updateItemOnSelectEvent()`, but they should not contain these values.